resource/inspire2019-test/rules.json

```diff
@@ -24,6 +24,83 @@
           "Identifier": "Session Catalog"
         }
       ]
+    },
+    {
+      "Identifiers": [
+        "Wristband Day Pass (internal use)", 
+        "Substituted Attendee Partner", 
+        "Substituted Attendee MS", 
+        "Non-Attending Manager",
+        "Ready Attendee"],
+      "Access": {
+        "SessionCatalog": true,
+        "MyIgnite": false,
+        "ScheduleBuilder": false,
+        "MobileApp": false,
+        "MeetingScheduler": false,
+        "MobileAppMessaging": false,
+        "LocationBasedMessaging": false,
+        "InPersonExperience": false
+      },
+      "VisibleAttendeeTypes": [],
+      "Reporting": {
+        "SessionCounts": false
+      },
+      "EvalAccess": [],
+      "SessionSetAccess": [
+        {
+          "ApplicableFrom": null,
+          "ApplicableTo": null,
+          "Identifier": "Session Catalog"
+        }
+      ]
+    },
+    {
+      "Identifiers": [
+        "Crew",
+        "Customer/Student - All Access",
+        "Customer/Student - Speaker Day Pass",
+        "MS - All Access",
+        "MS Corenote Speaker",
+        "MS Day Pass",
+        "MS Lounge & Booth Staff",
+        "MS Speaker Internal",
+        "Partner - All Access",
+        "Partner - Day Pass",
+        "Partner - Sponsor All Access",
+        "Partner - Sponsor Booth Staff",
+        "Press/Analyst",
+        "Speaker External",
+        "Speaker External Corenote",
+        "Speaker External Day Pass"
+      ],
+      "Access": {
+        "SessionCatalog": true,
+        "MyIgnite": true,
+        "ScheduleBuilder": true,
+        "MobileApp": false,
+        "MeetingScheduler": false,
```

Comment:
Pretty sure that these people can use the meeting scheduler and messaging features